tests/block_tools.py

```diff
@@ -256,7 +261,6 @@ async def setup_plots(self):
             await self.new_plot(
                 path=self.plot_dir / "not_in_keychain",
                 plot_keys=PlotKeys(G1Element(), G1Element(), None),
```

Comment:
```suggestion
                plot_keys=PlotKeys(G1Element(), G1Element(), None),
                exclude_plots=True,
```

This fixes the tests.